.pylintrc

```diff
@@ -1,2 +1,7 @@
 [MESSAGES CONTROL]
-disable=invalid-name, missing-function-docstring, missing-class-docstring, missing-module-docstring, unused-wildcard-import, cyclic-import, blacklisted-name, import-outside-toplevel
+disable=all
+enable=
+    C0121,C0200,C0209,C0321,C0325,C0415,
```

Comment:
Nits, thanks for cleaning this up, seems the old version with names is easier to identify?

Reply:
It might be, but this is adapted from `scripts/generate_pylint_tags.py`. IMO it is also straightforward to extract the check IDs from the pylint report.

Reply:
Ideally, we will change to `disable` pattern in the future.

Reply:
okay, thanks